SilentMoonsEnchantmentPatcher/Program.cs

```diff
@@ -440,6 +441,8 @@ FormList GetFormList(string name)
                 })
                 .Where(x => x != null)
                 .Select(x => x!)
+                .GroupBy(x => x.AsLink())
```

Comment:
Hey!  Thanks for the PR!

I think this logic can be simplified to just be:
`.Distinct(x => x.FormKey)`

And while we're in the area, the 
```
.Where(x => x != null)
.Select(x => x!)
```
can be simplified to
`.NotNull()`